### ISSUE TITLE
Split imported Name into First & Last fields in Drupal

### ISSUE THREAD

**Issue by mfurdyk** [DoSomething/laravel-user-api#19]
Hi @mshmsh5000 and @sergii-tkachenko,

Our team was just noticing that imported users have their full name pulled into the First Name field which means neither the first or last name fields in the search work (because they look for exact matches)

Could you set up the API to split up the name when it imports into Drupal, and would it be difficult to run this same split on the existing users to migrate the names apart? Not high-priority but would be nice as we promote some upcoming campaigns to our existing audience.

Thanks!

Michael

**Comment by mshmsh5000** [comment]
Hi! This happens when they're imported from TIG to DS? Just want to understand the flow.

**Comment by mfurdyk** [comment]
Yes - it pulls the full name from TIG into just the First Name field on DS.

Michael Furdyk, Co-founder & Director of Innovation, TakingITGlobal
Empowering youth to understand and act on the world’s greatest challenges
http://profiles.tigweb.org/mfurdyk | +1 (416) 977-9363 x313 | @mfurdyk
